scripts/testing/ats.sh.in

```diff
@@ -14,31 +14,67 @@
 # BlueOS: lalloc 2 ./ats.sh
 # WSL: ./ats.sh
 
+Help()
+{
+    echo "Test Invocations"
+    echo "=========================="
+    echo "TOSS3: salloc -N2 ./ats.sh"
+    echo "BlueOS: lalloc 2 ./ats.sh"
+    echo "WSL: ./ats.sh"
+    echo "Options:"
+    echo "  --rebaseline = re-makes all baselines"
+    echo "  --help = displays this message"
+}
+
+
 # Variables used inside of the base test.ats and this script
 export ATS_SERAC_BIN_DIR=@SERAC_BIN_DIR@/bin
 export ATS_SERAC_REPO_DIR=@SERAC_REPO_DIR@
 export ATS_EXECUTABLE=@ATS_EXECUTABLE@
+export ATS_SERAC_REBASELINE=0
 
 if [ ! -f "$ATS_EXECUTABLE" ]; then
     echo "ERROR: Given ATS_EXECUTABLE does not exist: " $ATS_EXECUTABLE
     exit 1
 fi
 
```

Comment:
Can you add a test to make sure serac is present

Reply:
I ended up checking for both `ATS_SERAC_BIN_DIR` and `ATS_SERAC_REPO_DIR`.